README.md

```
@@ -1,4 +1,4 @@
-## 가상환경 세팅 
+## 가상환경 세팅
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)